application.yml

```diff
@@ -19,11 +21,6 @@ sm:
 cf:
   client:
     apiAddress: http://api.local.pcfdev.io
-    username: admin
-    password: admin
     skipSslValidation: false
     httpClient:
-      timeout: 6000ms
-  reg:
```

Comment:
fails to start, since currently as implemented you need both app. and reg. 2018-10-23T11:32:53.83+0300 [APP/PROC/WEB/0] ERR panic: error creating CF client: validate settings: missing host